crates/holochain_conductor_api/src/admin_interface.rs

```diff
@@ -20,7 +20,7 @@ use crate::{AppInfo, FullStateDump, RevokeAgentKeyPayload, StorageInfo};
 // and the enum variant on a key `type`, e.g.
 // `{ type: 'enable_app', data: { installed_app_id: 'test_app' } }`
 #[derive(Debug, serde::Serialize, serde::Deserialize, SerializedBytes)]
-#[serde(rename_all = "snake_case", tag = "type", content = "data")]
+#[serde(tag = "type", content = "value", rename_all = "snake_case")]
```

Comment:
Note that this one and some others had previously already `content = "data"` and are now changed to `content = "value"`.